crates/marlin_arkworks_backend/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 
 [dependencies]
 
-acvm = { path = "../acvm", features = ["bn254"] }
+acvm = { git = "https://github.com/noir-lang/noir", features = ["bn254"] }
```

Comment:
The acvm crate is now pointing to the whole noir crate?

Reply:
Nevermind, I didn't think cargo would select the crate within the whole directory, it should be fine.

Can you elaborate more on the reasoning for this PR though?

Reply:
Yeah I was worried about this too and separated the acvm entirely at first (although this may be worth to do in the future), but then found out that Cargo selects the crate within the whole directory. I give some more details in my quote reply to @guipublic.